Makefile

```diff
@@ -1,6 +1,6 @@
 CONTAINER_ENGINE ?= podman
 IMAGE_REGISTRY   ?= quay.io/opendatahub/notebooks
-IMAGE_TAG		 ?=	$(eval $(call generate_image_tag,IMAGE_TAG))
+IMAGE_TAG	 ?= $(eval $(call generate_image_tag,IMAGE_TAG))
```

Comment:
Since we are using it directly inside the bootstrap and base recipes, I think would be okay if we delete it from here

Reply:
Keeping this as discussed.